src/Comet/EnvironmentData.cs

```diff
@@ -101,14 +106,13 @@ public static class Navigation
 		}
 		public static class Slider
 		{
-			public const string TrackColor = "SliderTrackColor";
-			public const string ProgressColor = "SliderProgressColor";
-			public const string ThumbColor = "SliderThumbColor";
+			public const string TrackColor = "MinimumTrackColor";
```

Comment:
Why are the values changing?

Reply:
Cause the MAUI API changed, you know the Comet.SourceGenerator project, the Slider class output file is this.
```
				Microsoft.Maui.Graphics.Color Microsoft.Maui.ISlider.MinimumTrackColor => this.GetEnvironment<Microsoft.Maui.Graphics.Color>("MinimumTrackColor") ?? default;

				Microsoft.Maui.Graphics.Color Microsoft.Maui.ISlider.MaximumTrackColor => this.GetEnvironment<Microsoft.Maui.Graphics.Color>("MaximumTrackColor") ?? default;

				Microsoft.Maui.Graphics.Color Microsoft.Maui.ISlider.ThumbColor => this.GetEnvironment<Microsoft.Maui.Graphics.Color>("ThumbColor") ?? default;

				Microsoft.Maui.IImageSource Microsoft.Maui.ISlider.ThumbImageSource => this.GetEnvironment<Microsoft.Maui.IImageSource>("ThumbImageSource") ?? default;

```